File: java/src/main/java/com/android/frame/mvp/AATest/activity/weather/WeatherView.java
package com.android.frame.mvp.AATest.activity.weather;

import com.android.frame.mvp.AATest.bean.WeatherBeanMvp;
import com.android.frame.mvp.IBaseView;

/**
 * Created by xuzhb on 2020/1/8
 * Desc:
 */
public interface WeatherView extends IBaseView {

    void showData(WeatherBeanMvp bean);

}

File: kotlin/src/main/java/com/android/frame/mvp/AATest/activity/weather/WeatherView.kt
package com.android.frame.mvp.AATest.activity.weather

import com.android.frame.mvp.AATest.bean.WeatherBeanMvp
import com.android.frame.mvp.IBaseView

/**
 * Created by xuzhb on 2020/1/4
 * Desc:
 */
interface WeatherView : IBaseView {

    //展示天气信息
    fun showData(bean: WeatherBeanMvp)

}